magentoerpconnect_pricelist_currency/model/sale.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2015 FactorLibre (http://www.factorlibre.com)
+#                  Hugo Santos <hugo.santos@factorlibre.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields
+from openerp.addons.connector.unit.mapper import mapping
+from openerp.addons.magentoerpconnect.backend import magento
+from openerp.addons.magentoerpconnect.sale import SaleOrderImportMapper, \
+    SaleOrderImporter
+
+
+class MagentoSaleOrder(models.Model):
+    _inherit = 'magento.sale.order'
+
+    magento_currency_id = fields.Many2one('res.currency', 'Magento Currency')
+
+
+@magento(replacing=SaleOrderImportMapper)
```

Comment:
Perhaps you could `replace` https://github.com/OCA/connector-magento/blob/8.0/magentoerpconnect/sale.py#L942-L948, but then your module would be incompatible with `magentoerpconnect_pricing` ( see https://github.com/OCA/connector-magento/blob/8.0/magentoerpconnect_pricing/sale.py#L27-L41)